crates/aptos-faucet/ts-client/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Aptos Tap TS client changelog
+
+All notable changes to the client will be captured in this file. This changelog is written by hand for now. It adheres to the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+**Note:** The client does not follow semantic version while we are in active development. Instead, breaking changes will be announced with each devnet cut. Once we launch our mainnet, the SDK will follow semantic versioning closely.
```

Comment:
Should this be updated? "Once we launch our mainnet"

Reply:
Ah yes, good call out.